Adyen/Core/Core Protocols/PaymentMethod.swift

```diff
@@ -30,6 +30,14 @@ public protocol PaymentMethod: Decodable {
     func buildComponent(using builder: PaymentComponentBuilder) -> PaymentComponent?
 }
 
+public extension PaymentMethod {
+
+    @_spi(AdyenInternal)
+    func buildComponent(using builder: PaymentComponentBuilder) -> PaymentComponent? {
+        nil
```

Comment:
Maybe a good place to use AssertionFailure? And (if possible) conditionally add this default conformance :)

Reply:
How about just using default implementation?
[edfa105](https://github.com/Adyen/adyen-ios/pull/1413/commits/edfa1058ef897ad6899bbe6e10f6f1671c98f603)